bundles/framework/mydata/Flyout.js

```diff
@@ -0,0 +1,229 @@
+import { Tabs } from 'antd';
+import { MyDataHandler } from './handler/MyDataHandler';
+import { MyViewsTab } from './view/MyViews//MyViewsTab';
+import { FlyoutContent } from './FlyoutContent';
+import { PublishedMapsHandler } from './handler/PublishedMapsHandler';
+import { MyViewsHandler } from './handler/MyViewsHandler';
+import { PublishedMapsTab } from './view/PublishedMaps/PublishedMapsTab';
+
+/**
+ * @class Oskari.mapframework.bundle.personaldata.Flyout
+ */
+Oskari.clazz.define('Oskari.mapframework.bundle.personaldata.Flyout',
+
+    /**
+     * @method create called automatically on construction
+     * @static
+     * @param {Oskari.mapframework.bundle.personaldata.PersonalDataBundleInstance} instance
+     *     reference to component that created the flyout
+     */
+
+    function (instance) {
+        this.instance = instance;
+        this.container = null;
+        this.state = null;
+
+        this.template = null;
+        this.templateTabHeader = null;
+        this.templateTabContent = null;
+        this.uiHandler = new MyDataHandler(() => this.update());
+        this.publishedMapsHander = new PublishedMapsHandler(() => this.update(), this.instance);
+        this.myViewsHandler = new MyViewsHandler(() => this.update());
+        this.element = null;
+    }, {
+        /**
+         * @method getName
+         * @return {String} the name for the component
+         */
+        getName: function () {
+            return 'Oskari.mapframework.bundle.personaldata.Flyout';
+        },
+        /**
+         * @method setEl
+         * @param {Object} el
+         *     reference to the container in browser
+         * @param {Number} width
+         *     container size(?) - not used
+         * @param {Number} height
+         *     container size(?) - not used
+         *
+         * Interface method implementation
+         */
+        setEl: function (el, flyout, width, height) {
+            this.container = el[0];
+            if (!jQuery(this.container).hasClass('personaldata')) {
+                jQuery(this.container).addClass('personaldata');
+            }
+            if (!flyout.hasClass('personaldata')) {
+                flyout.addClass('personaldata');
+            }
+        },
+        /**
+         * @method startPlugin
+         *
+         * Interface method implementation, assigns the HTML templates that will be used to create the UI
+         */
+        startPlugin: function () {
+            /* jQuery('#oskari-profile-link').on('click', function () {
+                me.instance.openProfileTab();
+                return false;
+            }); */
+
+            this.addTab(
+                'account',
+                Oskari.getMsg('PersonalData', 'tabs.account.title'),
+                AccountTab,
+                {
+                    getState: () => ({ user: Oskari.user() }),
+                    getController: () => ({ changeInfoUrl: () => Oskari.urls.getLocation('profile') }),
```

Comment:
We should pass both user and url in state to emphasize getters are not provided by controllers. Controller can be a no-op function here:
```suggestion
                    getState: () => ({
                        user: Oskari.user(),
                        changeInfoUrl: Oskari.urls.getLocation('profile')
                     }),
                    getController: () => null),
```